src/AppFrame.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render, screen } from "./testUtils";
+import { AppFrame } from "./AppFrame";
+import userEvent from "@testing-library/user-event";
+
+describe("AppFrame", () => {
+  test("Should switch to French", async () => {
+    render(<AppFrame />);
+    const francais = await screen.findByText("Français");
+    userEvent.click(francais);
+    await screen.findByText("Langue"); // Look for lowercase here, CSS is used to change text to uppercase
```

Comment:
No action required, but there is a Chrome extension called "Testing Playground" that lets you inspect an element in devtools and supplies the best possible query to find an element with testing-library. In this case it would have suggested `/langue/i` - you can use regex to query elements and ignore case.

Reply:
I only bothered to test French here and not any other languages here. Just wanted to write a basic unit test here to test that the language toggle works. New to this and testing-library is rather slow

Reply:
Yes, and this was a good test. Just wanted to point you in the direction of a helpful tool. All libs that mount  components to a virtual DOM are unfortunately somewhat slow.